### PR TITLE
tests: add details to some tests

### PR DESCRIPTION
**Pull request by MiguelPires**
Add details to some spread tests.

### CODE REVIEW

File: tests/main/interfaces-system-dbus/task.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 summary: Ensure that the dbus interface works on the system bus.
 
+details: |
+   Check that the dbus interface can be used to access the system
+   bus for specific methods and that they cannot be access if the
```
Comment (zyga) [inline tests/main/interfaces-system-dbus/task.yaml:5]:
```suggestion
   bus for specific methods and that they cannot be accessed if the
```

File: tests/main/services-stop-mode-sigkill/task.yaml [hunk 2]
```diff
@@ -1,5 +1,10 @@
 summary: "Check that refresh-modes sigkill works"
 
+details: |
+    Check that a snap service can be configured to be stopped with the sigterm
```
Comment (zyga) [inline tests/main/services-stop-mode-sigkill/task.yaml:4]: Technically, those are not signals but a means to use signals to stop the service.

File: tests/main/services-stop-mode/task.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 summary: "Check that stop-modes works"
 
+details: |
+    Check that a snap service can configure what signal it receives when it's
```
Comment (zyga) [inline tests/main/services-stop-mode/task.yaml:4]:
```suggestion
    Check that a snap service can configure which signal it receives when it's
```

**Review by zyga — APPROVED** [review]
LGTM, but please rephrase the sigterm-all description and look at my other comment.

File: tests/main/services-stop-mode-sigkill/task.yaml (hunk 2 above)
Comment (MiguelPires) [reply tests/main/services-stop-mode-sigkill/task.yaml]: I've rephrased this now. Thank you

**Review by andrewphelpsj — APPROVED** [review]
Thank you!

**Pull request merged**